docs/CONNECTOR_PROTOCOL.md

```diff
@@ -82,29 +110,111 @@ This is our main communication index, used to communicate the connector's config
 #### Elasticsearch mappings for `.elastic-connectors`:
 ```
 "mappings" : {
+  "_meta" : {
+    "pipeline" : {
+      "default_extract_binary_content" : true,
+      "default_name" : "ent-search-generic-ingestion",
+      "default_reduce_whitespace" : true,
+      "default_run_ml_inference" : true
+    },
+    "version" : "1"
```

Comment:
What is this `version` used for?

Reply:
It's a standard thing to put in your index mappings to help version the mappings. Kibana and ent-search have been using this since the beginning.

Reply:
I think this is not documented as a special field in [metadata field](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-meta-field.html), do you mean it's something implemented by Enterprise Search?

Will the version be increased if the meta is updated (e.g. `default_run_ml_inference` is set to `false`)? Because I don't see it happening in [Kibana](https://github.com/elastic/kibana/blob/73f1878983113d061890a8648f0548ac46823ca5/x-pack/plugins/enterprise_search/server/lib/pipelines/update_default_pipeline.ts#L33-L36).

Or will the version be increased if we add more fields to meta?